# The current taret of a UAV
uint16 target_seq_no 
bool is_global
float64 x_lat
float64 y_long
float64 z_alt
bool reached


